xbuild/src/devices/imd.rs

```diff
@@ -127,6 +127,9 @@ impl IMobileDevice {
     pub fn arch(&self, device: &str) -> Result<Arch> {
         match self.getkey(device, "CPUArchitecture")?.as_str() {
             "arm64" | "arm64e" => Ok(Arch::Arm64),
+            "armv7" => Ok(Arch::Armv7),
+            "arm" => Ok(Arch::Arm),
+            "x86_64" => Ok(Arch::X64),
```

Comment:
This is iOS code, did you confirm that it returns `x86_64`?

Reply:
I honestly didn't know what this does in first place